grails-app/conf/Config.groovy

```diff
@@ -137,6 +137,8 @@ environments {
         grails.serverURL = "http://${localhostAddress}:8080/$appName"
         gogoduck.url = "http://${localhostAddress}:8300/go-go-duck"
         geonetwork.url = "https://catalogue-123.aodn.org.au/geonetwork"
+
+        grails.mail.port = com.icegreen.greenmail.util.ServerSetupTest.SMTP.port
```

Comment:
I want SES because it'll reduce by 1 the amount of services we depend on...

But yes, we can also start with sendgrid. Essentially it's just a simple data bag.